### DISCUSSION TITLE
how to get the .nii file of the image from the .h5 file,  and  where is the demographic information

### DISCUSSION THREAD

**Discussion by ZhyCong** [facebookresearch/fastMRI#341] [Q&A]
Hello, is there a way to get the .nii file of the image from the .h5 file? Where is the demographic information? Thank you very much for your answers

**Comment by mmuckley** [comment]
Hello @ZhyCong, I don't think any demographic information was released with the fastMRI knee or brain data. I am less up to date on newer releases, like prostate.

As for constructing a `.nii` file, it depends a bit on your goal and what downstream software you want to use. You can of course make a simple `.nii` file by manually filling in metadata fields, but that may cause issues with downstream software if it depends on the metadata.

Hopefully if some other users have gone this path, they can comment here with their experience.
